src/pvrclient-nextpvr.cpp

```diff
@@ -851,29 +909,21 @@ bool cPVRClientNextPVR::IsServerStreamingLive(bool log)
     return true;
   }
   if (log)
-    kodi::Log(ADDON_LOG_ERROR, "Unknown live streaming state %d %d %d", m_nowPlaying, m_recordingBuffer->GetDuration(), !m_livePlayer);
+    kodi::Log(ADDON_LOG_ERROR, "Unknown live streaming state %d %d %d", m_nowPlaying, m_multistreamRecording.size(), !m_livePlayer);
   return false;
 }
 
-bool cPVRClientNextPVR::IsServerStreamingRecording(bool log)
+bool cPVRClientNextPVR::IsServerStreamingRecording(int64_t streamId, bool log)
 {
-  if (m_nowPlaying == Recording && m_recordingBuffer->GetDuration() > 0)
+  if (m_nowPlaying == Recording && m_multistreamRecording.size() > 0)
```

Comment:
`std::lock_guard<std::recursive_mutex> lock(m_multistreamRecording);` prior to accessing map.

No need to stop the lock, happens automatically at the end block